tasks/install-drivers.yml

```diff
@@ -0,0 +1,21 @@
+---
+
+- name: install | pro-drivers (download)
+  get_url:
+    url: "https://cdn.rstudio.com/drivers/7C152C12/installer/rstudio-drivers_2021.10.0_amd64.deb"
```

Comment:
i think this should be moved into vars and handle multiple architecture like here:
- https://github.com/Appsilon/ansible-rstudio-workbench/blob/main/vars/_focal.yml#L3

Reply:
That is a good point but look at those links:
- rstudio-drivers-2021.10.0-1.el7.x86_64.rpm
- rstudio-drivers_2021.10.0_amd64.deb

do you see how different they are?